frontend/src/host_orchestrator/api/v1/messages.go

```diff
@@ -208,3 +208,5 @@ type DisplayScreenshotResponse struct {
 	ScreenshotBytesBase64 string `json:"screenshot_bytes"`
 	ScreenshotMimeType    string `json:"screenshot_mime_type"`
 }
+
+type ScreenRecordingListResponse []string
```

Comment:
Use struct to represent top-level request and response objects: https://google.aip.dev/132#response-message